W-17361464 postrel polish duke (#325)
mulesoft/docs-monitoring: modules/ROOT/pages/logs.adoc

. If applicable, enter a value. +
The following image shows an example of using a search filter to search for all logs that have a timestamp that falls between the given dates.
+

> image::log-filter-timestamp.png["Filter query example"]

Author: ```suggestion
image::log-filter-timestamp.png["A form to add a filter for timestamps"]
```

By default, the log entries in your log search results are abbreviated. You can expand and view the log message and details as a table or in JSON format.

> image::log-expanded.png["Expanded log message"]

Author: ```suggestion
image::log-expanded.png["A log file entry"]
```

+
. View the extended log in tabular or JSON format.
+

> image::logs-view-extended.png["Extended logs"]

Author: ```suggestion
image::logs-view-extended.png["Extended logs example"]
```

+
. Load the number of logs that you want to see before and after the selected log, or use the default.
+

> image::logs-set-surrounding.png["Surrounding logs view"]

Author: ```suggestion
image::logs-set-surrounding.png["Surrounding logs view example"]
```

Click the column icon for a filter field, for example:

> image::log-filter-add-field.png["Log filter column addition"]

Author: ```suggestion
image::log-filter-add-field.png["Log filter column example"]
```
